backend/Player/serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from .models import Player
+from Match.models import Match
+from User.models import User
+
+class PlayerSerializer(serializers.ModelSerializer):
+	user = serializers.PrimaryKeyRelatedField(queryset=User.objects.all())
+	match = serializers.PrimaryKeyRelatedField(queryset=Match.objects.all())
```

Comment:
we use foreing keys elsewhere as user_id, match_id

Reply:
yes, but look at the latest presentation; I recommend we switch the previous user_ids etc. into simply User for the reasons specified in the presentation.